### PR TITLE
Description of responsive design breakpoints (#resolves #801)

### LINKED ISSUE #801: describe responsive design guidelines

### PR DESCRIPTION
**Pull request by magul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,3 +71,8 @@ Working agreement:
 * Regression lasts for one week and all of the regression bugs, that have been found, should be fixed with highest priority in week 6.
 * Afterwards application is to be tested and approved by the client. 
 * Sign-off from the client means that we push the code to the production after week 6.
+
+
+## Responsive desing breakpoints
```
Comment (w1stler) [inline README.md]: design

File: README.md [hunk 2]
```diff
@@ -71,3 +71,8 @@ Working agreement:
...
+
+Angular implementation use extensively Bootstrap 4 - we will comply with [standard Bootstrap 4 breakpoints described on its website](https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints).
```
Comment (w1stler) [inline README.md]: it's website

Comment (w1stler) [reply README.md]: Agree, but I am not sure if we want to keep all 4 breakpoints. We might discuss that with @Bosmanfrx and @TrebuhD .

Comment (magul) [reply README.md]: @w1stler that's not `it is` - it and adjective that describe a website `of it` (see: https://pl.pons.com/t%C5%82umaczenie/angielski-polski/its)

Generally we don't need to create separate media query for extra small devices, that enough to remember about them and to see if volontulo looks sane on them.

File: README.md (hunk 1 above)
Comment (magul) [reply README.md]: done

**Pull request merged**